src/aat/resources/features/F-066/F-066.feature

```diff
@@ -1,69 +1,96 @@
+#=======================================================================
 @F-066
 Feature: F-066: Retrieve a start event trigger by ID for dynamic display
+#=======================================================================
```

Comment:
Could you please change the capitalisation of all the feature names like below: (First letters in capital except for 'a', 'the', 'in', 'of', etc. )
`Feature: F-066: Retrieve a Start Event Trigger by ID for Dynamic Display`